SPECS/initramfs/initramfs.spec

```diff
@@ -88,15 +88,17 @@ echo "initramfs (re)generation" %* >&2
 cat > /dev/null \
 if [ -f %{_localstatedir}/lib/rpm-state/initramfs/regenerate ]; then \
     echo "(re)generate initramfs for all kernels," %* >&2 \
-    mkinitrd -q \
+    mkinitrd --preload megaraid_sas --preload megaraid -q \
     mv /boot/initrd.img-*mshv* /boot/efi/ \
 elif [ -d %{_localstatedir}/lib/rpm-state/initramfs/pending ]; then \
     for k in `ls %{_localstatedir}/lib/rpm-state/initramfs/pending/`; do \
         echo "(re)generate initramfs for $k," %* >&2 \
         if [[ $k == *mshv* ]]; then \
```

Comment:
what's mshv? 
Mainly trying to understand if preloading the megaraid drivers there makes sense.

Reply:
kernel-mshv. This PR adds megaraid* drivers regardless of the kernel variant installed.